### ISSUE TITLE
[BUG] Launching session does not work for Python scripts, only on notebooks

### ISSUE THREAD

**Issue by rossdanlm** [Arize-ai/phoenix#2710]
**Describe the bug**
Cannot launch session on localhost for Python script (non-notebook) when following the tracing tutorial doc: https://docs.arize.com/phoenix/tracing/llm-traces-1. This works for Jupyter notebooks, but in general prefer scripts so wish that could work too.

**To Reproduce**
1. Follow doc commands
2. Output will say to view on localhost
3. Localhost does not show session view, "site can't be reached"

**Expected behavior**
Should see session view

**Screenshots**
https://github.com/Arize-ai/phoenix/assets/151060367/7057e3e4-f4d8-4012-b78b-0a4fe1b60382

**Environment (please complete the following information):**
 - OS: MacOS
 - Notebook Runtime: Python script (note that I got this to work on Jupyter Notebooks, just wondering why a script won't work?)
 - Browser chrome
 - Version 3.17.1

**Additional context**
Add any other context about the problem here (e.x. a link to a colab)

**Comment by axiomofjoy** [comment]
Hey @rossdanlm, thanks for trying Phoenix! `phoenix.launch_app` is non-blocking and is tied to the parent process, so when the parent process dies, so will your session. It is primarily intended for use in notebooks. If you wish to your code from a script, we recommend running Phoenix from the command line with `python -m phoenix.server.main serve`.

**Comment by axiomofjoy** [comment]
Closing this issue for now, feel free to re-open.

**Comment by rossdanlm** [comment]
Thanks for the quick reply! Would love to have this info added to the docs (even just an aside (click here for instructions on how to run from command-line instead of notebooks)) so others can get unblocked when getting started without relying on notebooks!

**Comment by axiomofjoy** [comment]
> Thanks for the quick reply! Would love to have this info added to the docs (even just an aside (click here for instructions on how to run from command-line instead of notebooks)) so others can get unblocked when getting started without relying on notebooks!

Good callout! I created a separate issue [here](https://github.com/Arize-ai/phoenix/issues/2711).
